ps2/stdint.h

```diff
@@ -0,0 +1,48 @@
+/* RetroArch - A frontend for libretro.
+ * Copyright (C) 2010-2018 - Francisco Javier Trujillo Mata - fjtrujy
+ *
+ * RetroArch is free software: you can redistribute it and/or modify it under the terms
```

Comment:
Ditto.

Reply:
Ditto xD

Reply:
@fjtrujy will take out the license text and that will resolve the issue.